### DISCUSSION TITLE
how config  the crash reporting

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1799] [Q&A] [answered]
i want open the .bim file ( create from iTwin Snapshot)
web app should use a shared tile cache service instead (The .tiles tile cache),
so i want how config the  shared tile cache service .

i set this,but the d:\temp3 dictory is empty after open the .bim file by web brower, and create .Tiles exist.

const config = new IModelHostConfiguration();
config.crashReportingConfig={
  crashDir: "d:\\\\temp3",  
  enableCrashDumps:true,
  enableNodeReport:true,
};  
await IModelHost.startup(config);

**Comment by pmconne** [comment, marked as answer]
Crash reporting is about logging information when the backend process crashes.
It has absolutely nothing to do with tiles.
An empty crashDir is a good thing - it means your backend did not crash.

**Reply by paprophet** [reply]
thank you 
 the Crash reporting  -- log the Crash  of the errors in Web app backends, services, and agents
